pool.go

```diff
@@ -217,6 +298,15 @@ func (p *pool) fetchBlockWith(ctx context.Context, c cid.Cid, with string) (blk
 		if recordIfContextErr(resourceTypeBlock, ctx, "fetchBlockWithLoop") {
 			return nil, ctx.Err()
 		}
+
+		// sample request for mirroring
+		if p.config.MirrorFraction > rand.Float64() {
+			select {
+			case p.mirrorSamples <- poolRequest{node: nodes[i], path: fmt.Sprintf("/ipfs/%s?format=raw", c), key: aff}:
```

Comment:
So we're converting a block request to a CAR request ? How does this work ? Will Nginx/L1 shim/Lassie just send back that one block in a CAR file ?

Reply:
since we ask for `format=raw` we get back directly the block data, not even a car here - i that's a good catch - yes the intention here was to get a car with the single block back